src/main/java/com/scriptbasic/executors/commands/CommandEndSelect.java

```diff
@@ -0,0 +1,15 @@
+package com.scriptbasic.executors.commands;
+
+import com.scriptbasic.api.ScriptBasicException;
+import com.scriptbasic.spi.Interpreter;
+
+public class CommandEndSelect
+	extends AbstractCommand
+{
+
+	@Override
+	public void execute(Interpreter interpreter) throws ScriptBasicException {
+		// do nothing		

```

Comment:
For me this comment is meaningless. I can see that this does nothing: there are no Java commands in it. If you feel there is a need for some comment then you can describe the fact that the "END SELECT" command does not need to execute in the JavaDoc of the method.